sources/api/schnauzer/src/helpers.rs

```diff
@@ -646,6 +783,20 @@ fn ecr_registry<S: AsRef<str>>(region: S) -> String {
     format!("{}.dkr.ecr.{}.amazonaws.com", registry_id, region)
 }
 
+/// Calculates and returns the amount of CPU to reserve
+fn cpu_reserve(num_cores: usize) -> String {
```

Comment:
If you take the suggestion to rename the template-level helpers, this could be renamed to `kube_cpu_helper` so it doesn't read like it applies universally.